test/utils/elasticsearch.go

```diff
@@ -0,0 +1,207 @@
+package utils
```

Comment:
Consider placing in an `elasticsearch` package so you could import a call functions like 'elasticsearch.CreateCR`

Reply:
Unfortunately, we have many places with packages refering to elasticsearch. I would reserve it for production code. The e2e test util code is manageable enough in one utils package and `utils.CreateElasticsearchCR` is indicative enough.